polyfills.ts

```diff
@@ -1,20 +1,19 @@
 // Random values must be imported first for security.
+import "@ethersproject/shims";
 import "@walletconnect/react-native-compat";
 import "react-native-get-random-values";
-import "@ethersproject/shims";
 
-import "text-encoding";
 import "@azure/core-asynciterator-polyfill";
-import "react-native-polyfill-globals/auto";
 import { Crypto as WebCrypto } from "@peculiar/webcrypto";
 import "@stardazed/streams-polyfill";
-import ViewReactNativeStyleAttributes from "react-native/Libraries/Components/View/ReactNativeStyleAttributes";
```

Comment:
Are you sure we don't need this any more?

Reply:
From what I saw this is not needed anymore since `scaleY` is already in `View` component